src/cloudformation_cli_python_lib/recast.py

```diff
@@ -14,7 +14,7 @@ def recast_object(
     if not isinstance(json_data, dict):
         raise InvalidRequest(f"Can only parse dict items, not {type(json_data)}")
     # if type is Any, we leave it as is
-    if cls == typing.Any:
+    if cls is typing.Any:
```

Comment:
Made this change because of pylint error,
W0143: Comparing against a callable, did you omit the parenthesis? (comparison-with-callable)